packages/sitecore-jss-angular/src/components/router-link.directive.spec.ts

```diff
@@ -164,6 +164,19 @@ describe('<a *scRouterLink />', () => {
     expect(rendered.nativeElement.target).toBe('_blank');
     expect(rendered.nativeElement.title).toBe('footip');
   });
+
+  it('should strip query string from href', () => {
+    const field = {
+      href: '/lorem?sc_site=lorem_ipsum',
+      text: 'ipsum',
+    };
+    comp.field = field;
+    field.href = stripQueryParams(field.href);
+    fixture.detectChanges();
+
+    const rendered = de.query(By.css('a'));
+    expect(rendered.nativeElement.href).toBe('http://localhost:9876/lorem');
```

Comment:
Why is this an absolute URL with 'http://localhost:9876'?

Also, I was just looking through the tests we've got, and there really aren't any assertions on the href attribute ... Wonder if we should at least one for a normal case (without query parm)? The "should render all value attributes" test seems like a good candidate.

Reply:
> Why is this an absolute URL with 'http://localhost:9876'?

That's what was coming back from the test when it was failing, I thought it was strange too.

Yeah that seems worth adding.